src/functionaltests/java/com/ericsson/ei/utils/HttpRequests.java

```diff
@@ -0,0 +1,146 @@
+package com.ericsson.ei.utils;
+
+import com.ericsson.ei.controller.model.SubscriptionResponse;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.io.FileUtils;
+import org.apache.http.client.methods.*;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.io.*;
+
+
+public class HttpRequests {
+    private static final Logger LOGGER = LoggerFactory.getLogger(HttpRequests.class);
+    private int port;
+    private CloseableHttpClient client = HttpClientBuilder.create().build();
+    static JSONArray jsonParams = null;
+    private ObjectMapper mapper = new ObjectMapper();
+
+
+    public HttpRequests(int applicationPort) {
+        port = applicationPort;
+    }
+
+    /**
+     * Create a HTTP GET request
+     *
+     * @param url
+     *
+     * */
+    public ResponseEntity<String> makeHttpGetRequest(String url) {
+        ResponseEntity<String> response = null;
+
+        HttpGet httpGet = new HttpGet(url);
+        response = getResponse(httpGet);
+
+        return response;
+    }
+
+
+    /**
+     * Create a POST request
+     *
+     * @param url
+     *      Where to send request
+     * @param filePath
+     *      Location to json file containing subscription
+     * */
+    public ResponseEntity<String> makeHttpPostRequest(String url, String filePath) {
+        ResponseEntity<String> response = null;
+        String fileContent = "";
+
+        try {
+            fileContent = FileUtils.readFileToString(new File(filePath), "UTF-8");
+        } catch(IOException e) {
+            LOGGER.error(e.getMessage(), e);
+            System.out.println("ERROR: " + e.getMessage());
+        }
+
+        try {
+            jsonParams = new JSONArray(fileContent);
+        } catch(JSONException e) {
+            LOGGER.error(e.getMessage(), e);
+            System.out.println("ERROR: " + e.getMessage());
```

Comment:
Is println needed here when log.error with same message is already defined,, or doesn't log error always print to console?